Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsVC.swift

```diff
@@ -87,9 +87,10 @@ open class ChatMessageActionsVC: _ViewController, ThemeProvider {
     open var messageActions: [ChatMessageActionItem] {
         guard
             let currentUser = messageController.dataStore.currentUser(),
-            let message = message,
-            message.isDeleted == false
```

Comment:
why do we remove this condition?

Reply:
I've reverted this. This is a bit redundant, but better to keep it for now 👍